locale/fa/docs/guides/backpressuring-in-streams.md

```diff
@@ -60,7 +60,7 @@ In one scenario, we will take a large file (approximately ~9gb) and compress it
 using the familiar [`zip(1)`][] tool.
 
 ```
```

Comment:
Personally I'd mark those who lack a lang in this patch as `bash` so that it's consistent and so that they can later be targeted with CSS. Now it's plain text.

Reply:
Yeah, those would get flagged if/when MD040 gets enabled again